src/billing/Workspaces/Workspaces.tsx

```diff
@@ -9,11 +12,19 @@ import {
   isGoogleBillingProject,
 } from 'src/billing-core/models';
 import { ariaSort, HeaderRenderer } from 'src/components/table';
+import { Ajax } from 'src/libs/ajax';
```

Comment:
You may want to narrow Ajax usage, similar to what Miguel has been doing with recent PRs ([example](https://github.com/DataBiosphere/terra-ui/pull/5146) if you haven't reviewed one yet). Fixing existing instances is out of scope, but if you are adding new ones it would probably be better to add the narrow versions.

Reply:
Done: https://github.com/DataBiosphere/terra-ui/pull/5165/commits/9a4ef801a9fa8e2844b531ccff4f86a217d3fae8